### PR TITLE
Add russian translation of desktop file

### PR DESCRIPTION
**Pull request by greno4ka**

### CODE REVIEW

File: docs/desktopEntry/package/flameshot.desktop [hunk 1]
```diff
@@ -43,6 +46,7 @@ Name[zh_CN]=进行截图
 Name[pl]=Zrzut ekranu
 Name[fr]=Prendre une capture d'écran
 Name[ja]=スクリーンショットを撮る
+Name[sr]=Сделать скриншот
```
Comment (alok1111) [inline docs/desktopEntry/package/flameshot.desktop]: Maybe "Name[ru]"?

File: docs/desktopEntry/package/flameshot.desktop [hunk 2]
```diff
@@ -33,6 +35,7 @@ Name[zh_CN]=配置
 Name[pl]=Konfiguruj
 Name[fr]=Configurer
 Name[ja]=設定
+Name[sr]=Настройки
```
Comment (alok1111) [inline docs/desktopEntry/package/flameshot.desktop]: Maybe "Name[ru]"?

**Comment by greno4ka** [comment]
Oh noo... That ctrl+c, ctrl+v. Thanks a lot. I'll fix this PR soon.

**Comment by greno4ka** [comment]
Fixed. Thanks.

**Comment by lupoDharkael** [comment]
Thanks for the contribution!

**Pull request merged**